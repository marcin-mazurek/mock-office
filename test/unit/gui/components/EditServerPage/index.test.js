import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import EditServerPage from '../../../../../src/lib/gui/components/EditServerPage';

describe('EditServerPage', () => {
  test('default snapshot', () => {
    const wrapper = shallow(<EditServerPage params={{ id: 'id' }} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
