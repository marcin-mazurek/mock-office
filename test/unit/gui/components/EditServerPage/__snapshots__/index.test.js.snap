// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditServerPage default snapshot 1`] = `
<div
  className="edit-server-page"
>
  <Scrollbars
    autoHeight={false}
    autoHeightMax={200}
    autoHeightMin={0}
    autoHide={false}
    autoHideDuration={200}
    autoHideTimeout={1000}
    hideTracksWhenNotNeeded={false}
    renderThumbHorizontal={[Function]}
    renderThumbVertical={[Function]}
    renderTrackHorizontal={[Function]}
    renderTrackVertical={[Function]}
    renderView={[Function]}
    tagName="div"
    thumbMinSize={30}
    universal={false}
  >
    <div
      className="edit-server-page__form"
    >
      <Connect(ReduxForm)
        serverId="id"
      />
    </div>
  </Scrollbars>
</div>
`;
